README.md

```diff
@@ -2,11 +2,15 @@
 
 [![Documentation Status](https://readthedocs.org/projects/microsoft-recommenders/badge/?version=latest)](https://microsoft-recommenders.readthedocs.io/en/latest/?badge=latest)
 
-## What's New (June 21, 2021)
+## What's New 
+### (August 23, 2021)
 
-We have a new release [Recommenders 0.6.0](https://github.com/microsoft/recommenders/releases/tag/0.6.0)!
+We have a new release [Recommenders 0.7.0](https://github.com/microsoft/recommenders/releases/tag/0.7.0)!
```

Comment:
> We have a new release [Recommenders 0.7.0](https://github.com/microsoft/recommenders/releases/tag/0.7.0)!

Maybe we should wait to add this link until we have the tag itself. Based on past experiences this process could take several weeks. I think it would be weird for users to go to the repo and see a link that doesn't work. What do you think?

Reply:
Ok, then probably this PR should be merged after the tag is created.